airflow/kubernetes/pod.py

```diff
@@ -37,6 +37,10 @@ def has_limits(self):
     def has_requests(self):
         return self.request_cpu is not None or self.request_memory is not None
 
+    def __str__(self):
+        return str({'request': {'memory': self.request_memory, 'cpu': self.request_cpu},
```

Comment:
`self.request_memory` can be None, may be check first?
same for `self.limit_memory`

Reply:
true, the test covers this, `None` to string is `'None'`